lib/beaker/hypervisor/openstack.rb

```diff
@@ -197,15 +197,15 @@ def provision
 
       @hosts.each do |host|
         ip = get_ip
-        hostname = ip.ip.gsub '.','-'
-        host[:vmhostname] = hostname+'.rfc1918.puppetlabs.net'
-        @logger.debug "Provisioning #{host.name} (#{host[:vmhostname]})"
+        host[:vmhostname] = ip.ip.gsub! '.','-'
```

Comment:
No Kev, you are right and a good reviewer :)  I ported this from a 2.51.0 tree and I missed this change.  Branch updated to take it into account